core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -210,7 +215,10 @@ object ConsumerPerformance {
    
     val useNewConsumer = options.has(useNewConsumerOpt)
     
-    val props = new Properties
+    val props = if (options.has(consumerConfigOpt))
+      Utils.loadProps(options.valueOf(consumerConfigOpt))
```

Comment:
I do completely agree. However, I think all the tools currently do it this way and so I have kept it consistent. I will raise another JIRA to fix all the tools.